static/js/compose_actions.js

```diff
@@ -67,11 +67,11 @@ exports.set_focus = function (msg_type, opts) {
 function show_box(msg_type, opts) {
     if (msg_type === "stream") {
         $('#private-message').hide();
-        $('#stream-message').show();
+        $('#stream-message').show().css({display: 'flex'});
         $("#stream_toggle").addClass("active");
         $("#private_message_toggle").removeClass("active");
     } else {
-        $('#private-message').show();
+        $('#private-message').show().css({display: 'flex'});
```

Comment:
Why do you need these here?  It seems weird to need to be adding inline CSS in the JS when we have a template...